.github/workflows/go.yml

```diff
@@ -235,6 +235,13 @@ jobs:
         with:
           name: k0s
 
+      - name: Download latest release
+        if: matrix.smoke-suite == 'check-ap-ha3x3'
+        run: |
+          RELEASE_VERSION=`curl --silent "https://api.github.com/repos/k0sproject/k0s/releases/latest" | grep '"tag_name":' |  sed -E 's/.*"([^"]+)".*/\1/'`
```

Comment:
Isn't the query against `/latest` sensitive to the ordering of releases in the releases page?  Does this need something like `k0s_sort`?

Reply:
Fixed